packages/cozy-client/src/CozyClient.js

```diff
@@ -605,6 +606,28 @@ client.query(Q('io.cozy.bills'))`)
     return this.mutate(this.getDocumentSavePlan(doc), mutationOptions)
   }
 
+  /**
+   * Saves multiple documents in one batch
+   * - Can only be called with documents from the same doctype
+   * - Does not support automatic creation of references
+   *
+   * @param  {CozyClientDocument[]} docs
+   * @param  {Object} mutationOptions
+   * @returns {Promise<void>}
+   */
+  async saveAll(docs, mutationOptions = {}) {
+    const doctypes = Array.from(new Set(docs.map(x => x._type)))
+    if (doctypes.length !== 1) {
+      throw new Error('saveAll can only save documents with the same doctype')
+    }
+    const ret = every(await Promise.all(docs.map(d => this.schema.validate(d))))
+    if (ret !== true) throw new Error('Validation failed for at least one doc')
```

Comment:
Shouldn't we log something? To help the debug in the app's land? To know the concerned document(s)?

Reply:
Done in 8c5a895c